vignettes/ci-providers.Rmd

```diff
@@ -61,6 +61,55 @@ If Java support is required, add the following for macOS runners:
 For Linux, add `sudo R CMD javareconf` to stage "[Linux] Prepare".
 We currently do not support Java on Windows.
 
+### ccache
+
+If you have a huge dependency chain and compiling many packages from source (especially on R-devel), `ccache` can help to speed up package installation.
+It is recommended once your dependency installation time is higher than 30 minutes.
+
+Once the `ccache` cache is build, compiled code will almost run instantly.
+The `ccache` cache itself is only invalidated once a month.
+This means package installation can make use of the cache in 29/30 days in a month.
+
+The downside is that `ccache` needs to be installed and configured.
+This happens in every run, i.e. also in runs in which `ccache` is not used because a package cache already exists.
+Installation can take up to 1 min, depending on the platform.
+Note that `ccache` won't be used on Windows since only binaries are used on this platform.
```

Comment:
Would it even work?

Reply:
If there would be a compiler, it would.

But this and the burden of installing `ccache` in the first place is not worth it.